File: examples/python/tasks/translation/translation.py
from bytez import Bytez

client = Bytez("YOUR BYTEZ KEY HERE")

input_text = "Hello, how are you? Beautiful day today, isn't it?"

# Load and run the model
model = client.model("Helsinki-NLP/opus-mt-en-zh")
model.load()

result = model.run(input_text)

# Extract the translation text
output = result["output"]
translation_text = output[0]["translation_text"]

print(translation_text)

File: examples/javascript/tasks/translation/translation.mjs
import Bytez from "bytez.js";

const client = new Bytez("YOUR BYTEZ KEY HERE");

const inputText = "Hello, how are you? Beautiful day today, isn't it?";

const model = client.model("Helsinki-NLP/opus-mt-en-zh");

await model.load();

const { output: [{ translation_text }] } = await model.run(inputText);

console.log(translation_text);
